packages.yml

```diff
@@ -1,6 +1,3 @@
 packages:
   - package: fivetran/hubspot_source
```

Comment:
FYI this needs to be pointing to the newer version of dbt_hubspot_source

Reply:
updated